src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +42,77 @@
     margin: 0;
 }
 
+.node-display-name {
+    font-weight: bold;
+    width: 134px;
+}
+
+.node-timestamp {
+    font-size: 1.1rem;
+    width: 134px;
+}
+
+.clock-icon {
+    /* We want it to be on top of everything except the sidebar */
+    z-index: 97;
+    margin-left: 40px;
+    bottom: 18px;
+    position: absolute;
+    width: 15%
+}
+
+.wait-time {
+    position: absolute;
+    right: -20px;
+    top: 100px;
+}
+
+.lengthened {
+    display: none;
+}
+
+.shortened {
+    display: initial;
+}
+
 @media (min-width: 992px) {
     .node-siblings-column {
         /* Leave a larger gap between the main node and the siblings when on a tablet or larger */
         padding-left: 2.5rem;
     }
+
+    .node-display-name {
+        font-weight: bold;
+        width: 209px;
+    }
+
+    .node-timestamp {
+        font-size: 1.1rem;
+        width: 209px;
+    }
+
+    .clock-icon {
```

Comment:
Most of this CSS is repeated from above. Could you please remove the duplicates?